.github/renovate.json

```diff
@@ -126,7 +126,7 @@
         "^latest\\/base.yml$"
       ],
       "matchStrings":[
-        "# renovate: datasource=(?<datasource>github-releases) depName=(?<depName>.*?)\n.*?docker: '5:(?<currentValue>.*?)'"
+        "# renovate: datasource=(?<datasource>github-releases) depName=(?<depName>.*?)\n.*?docker: '5:v(?<currentValue>.*?)'"
```

Comment:
That doesn't make any sense either. This now matches the v in the file which is not present there. The match string is correct as it is.